### PR TITLE
Feature/oe 5522

### PR DESCRIPTION
**Pull request by vetusko**
AdminListAutocomplete features

### CODE REVIEW

File: protected/assets/js/oeadmin/listAutocomplete.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        // .filterfieldselect: all filter fields
+        var saveParams = $('.ui-autocomplete-input').attr('name') + "=" + itemId;
+        $('.filterfieldselect').each(function () {
+            // name: search[filterid][field_name][value]
+            fieldNameData = $(this).attr('name').split('[');
+            fieldName = fieldNameData[2].replace(']', '');
+            saveParams += "&" + fieldName + "=" + $(this).val();
+        });
+
+        console.log(saveParams);
```
Comment (petergallagher) [inline protected/assets/js/oeadmin/listAutocomplete.js]: Can you remove this console.log please

File: protected/views/admin/generic/listAutocomplete.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+					</tr>
+				<?php } ?>
+				</tbody>
+				<tfoot class="pagination-container">
+				<tr>
+					<td colspan="<?php echo count($admin->getListFields()) + 1; ?>">
+						<?php
+						$acFieldData = $admin->getAutocompleteField();
+						if ($acFieldData) {
+							//var_dump($acFieldData["jsonURL"]);
```
Comment (petergallagher) [inline protected/views/admin/generic/listAutocomplete.php]: Can you remove the var_dump

File: protected/assets/js/oeadmin/listAutocomplete.js (hunk 1 above)
Comment (vetusko) [inline protected/assets/js/oeadmin/listAutocomplete.js]: removed

File: protected/views/admin/generic/listAutocomplete.php (hunk 2 above)
Comment (vetusko) [inline protected/views/admin/generic/listAutocomplete.php]: removed

**Pull request merged**